exporters/otlp/otlptrace/otlptracehttp/options.go

```diff
@@ -0,0 +1,125 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otlptracehttp
+
+import (
+	"crypto/tls"
+	"time"
+
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/internal/otlpconfig"
+)
+
+const (
+	// defaultMaxAttempts describes how many times the driver
+	// should retry the sending of the payload in case of a
+	// retryable error.
+	defaultMaxAttempts int = 5
+	// defaultTracesPath is a default URL path for endpoint that
+	// receives spans.
+	defaultTracesPath string = "/v1/traces"
+	// defaultBackoff is a default base backoff time used in the
+	// exponential backoff strategy.
+	defaultBackoff time.Duration = 300 * time.Millisecond
+)
```

Comment:
With these unexported they will be hard to find in our documentation.

Reply:
@MrAlias suggestions fix that problem. These are now documented on their respective `With*` options.